Yosemite/Yosemite/Stores/AppSettingsStore.swift

```diff
@@ -178,8 +178,8 @@ public class AppSettingsStore: Store {
             setCouponManagementFeatureSwitchState(isEnabled: isEnabled, onCompletion: onCompletion)
         case .loadCouponManagementFeatureSwitchState(let onCompletion):
             loadCouponManagementFeatureSwitchState(onCompletion: onCompletion)
-        case .setFeatureAnnouncementDismissed(campaign: let campaign, remindLater: let remindLater, onCompletion: let completion):
-            setFeatureAnnouncementDismissed(campaign: campaign, remindLater: remindLater, onCompletion: completion)
+        case .setFeatureAnnouncementDismissed(campaign: let campaign, remindLater: let remindLater, remindAfter: let remindAfter, onCompletion: let completion):
```

Comment:
I would encapsulate `remindLater` and `remindLater` in one value, I don't think we need both here. If we have both we can give place to inconsistent states, e.g., `remindLater` false and `remindAfter` to 5 days. For instance, we can solve this as follows:

- We can have only `remindAfter`, where 0 means it shouldn't be reminded. Or,
- `remindAfter` can be an optional, being `nil` meaning it shouldn't be reminded. I prefer this one.

Reply:
Much cleaner indeed, changes here: https://github.com/woocommerce/woocommerce-ios/pull/8657/commits/217758491434831705a861af3c0582da903af916 Now we have an unique Optional parameter `remindAfterDays` rather than two.